### PR TITLE
feat(rails-user-auth)- create user backend

### PR DESCRIPTION
**Pull request by Rachella**

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+== README
```
Comment (h8rry) [inline README.rdoc]: use file extension `.md`. So `README.md`

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (h8rry) [inline app/helpers/static_pages_helper.rb]: If you don't need these helpers, you can delete them

File: app/assets/stylesheets/static_pages.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/*
```
Comment (h8rry) [inline app/assets/stylesheets/static_pages.css]: If you don't need these css files, you can delete them

File: app/assets/javascripts/users.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (h8rry) [inline app/assets/javascripts/users.js]: If you don't need these JS files, you can delete them

File: config.ru [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# This file is used by Rack-based servers to start the application.
```
Comment (h8rry) [inline config.ru:1]: You should follow the instructions under `Rails` instead of `Rack`

https://github.com/cyu/rack-cors

**Comment by h8rry** [comment]
The rest looks fine

**Comment by Rachella** [comment]
Okay, helper/css/js files removed. Readme updated. and Rack-cors config fixed. Thanks :smile:

**Comment by Rachella** [comment]
Okay, you should be able to see the config/application.rb file now..

**Comment by h8rry** [comment]
Looks good!

**Pull request merged**